src/components/accession2/DetailPanel.tsx

```diff
@@ -51,115 +56,129 @@ export default function DetailPanel(props: DetailPanelProps): JSX.Element {
+              {strings.COLLECTION_DATE}
+            </Grid>
+            <Grid item xs={gridRightSide}>
+              {accession.collectedDate}
+            </Grid>
+          </Grid>
+          <Grid item xs={12} sx={gridRowStyle}>
+            <Grid item xs={gridLeftSide} sx={categoryStyle}>
+              {strings.COLLECTOR}
+            </Grid>
+            <Grid item xs={gridRightSide}>
+              {accession.collectors && accession.collectors.length > 0 ? accession.collectors[0] : ''}
+            </Grid>
+          </Grid>
+          <Grid item xs={12} sx={gridRowStyle}>
+            <Grid item xs={gridLeftSide} sx={categoryStyle}>
+              {strings.COLLECTING_LOCATION}
+            </Grid>
+            <Grid item xs={gridRightSide}>
+              {accession.collectionSiteName}
+              {accession.collectionSiteLandowner ? (
+                <Typography>
+                  ({strings.OWNER}: {accession.collectionSiteLandowner})
+                </Typography>
+              ) : (
+                ''
+              )}
+              {accession.collectionSiteCity &&
+              accession.collectionSiteCountrySubdivision &&
+              accession.collectionSiteCountryCode
+                ? `${accession.collectionSiteCity}, ${accession.collectionSiteCountrySubdivision},
               ${accession.collectionSiteCountryCode}`
-              : ''}
-            {accession.collectionSiteCoordinates && accession.collectionSiteCoordinates.length > 0
-              ? accession.collectionSiteCoordinates?.map(
-                  (coordinate) => `${coordinate.longitude},  ${coordinate.latitude}`
-                )
-              : ''}
+                : ''}
+              {accession.collectionSiteCoordinates && accession.collectionSiteCoordinates.length > 0
+                ? accession.collectionSiteCoordinates?.map(
+                    (coordinate) => `${coordinate.longitude},  ${coordinate.latitude}`
+                  )
+                : ''}
+            </Grid>
+          </Grid>
+          <Grid item xs={12} sx={gridRowStyle}>
+            <Grid item xs={gridLeftSide} sx={categoryStyle}>
+              {strings.PLANT_AND_SITE}
+            </Grid>
+            <Grid item xs={gridRightSide}>
+              {`${strings.COLLECTED_FROM} ${accession.plantsCollectedFromMax}-${accession.plantsCollectedFromMin} ${strings.WILD_PLANT}`}
+              {accession.founderId ? <Typography>{`${strings.PLANT_ID}: ${accession.founderId}`}</Typography> : ''}
+              {accession.notes}
+            </Grid>
+          </Grid>
+          <Grid item xs={12} sx={gridRowStyle}>
+            <Grid item xs={gridLeftSide} sx={categoryStyle}>
+              {strings.PHOTOS}
+            </Grid>
+            <Grid item xs={gridRightSide} display='flex'>
+              {accession.photoFilenames?.map((file, index) => {
+                return (
+                  <Box paddingRight={theme.spacing(2)} key={`photo-${index}`}>
+                    <img
```

Comment:
I added a fragment at the top so that I could add the <Modal> and it looks like a lot of changes, but changes are here. Basically replacing the old anchor with opening the new modal